source/common/tracing/http_tracer_impl.cc

```diff
@@ -187,6 +187,11 @@ void HttpTracerUtility::finalizeUpstreamSpan(Span& span, const Http::HeaderMap*
   span.setTag(Tracing::Tags::get().HttpProtocol,
               AccessLog::AccessLogFormatUtils::protocolToString(stream_info.protocol()));
 
+  if (stream_info.upstreamHost() && stream_info.upstreamHost()->address()) {
```

Comment:
The upstream host must have an address so I don't think the second part of this if statement is needed.

Reply:
yep, removed